grails-app/views/download/show.gsp

```diff
@@ -5,106 +5,122 @@
 		<meta name="layout" content="main">
 		<g:set var="entityName" value="${message(code: 'download.label', default: 'Download')}" />
 		<title><g:message code="default.show.label" args="[entityName]" /></title>
+
+		<style type="text/css">
+		.glyphicon-list {
+			color: #333;
+		}
+		.glyphicon-home {
+			 color: #333;
+		 }
+		.glyphicon-pencil {
+			color: #333;
+		}
+
+        .row-grid  [class*="col-"] {
+               margin-bottom: 15px;         
+         }
+
+
+		</style>
 	</head>
 	<body>
 		<a href="#show-download" class="skip" tabindex="-1"><g:message code="default.link.skip.label" default="Skip to content&hellip;"/></a>
 		<div class="nav" role="navigation">
-			<ul>
-				<li><a class="home" href="${createLink(uri: '/')}"><g:message code="default.home.label"/></a></li>
-				<li><g:link class="list" action="list"><g:message code="default.list.label" args="[entityName]" /></g:link></li>
-				<li><g:link class="create" action="create"><g:message code="default.new.label" args="[entityName]" /></g:link></li>
-			</ul>
+			<div class="row">
+				<div class="col-md-2"><a href="${createLink(uri: '/')}"><i class="glyphicon glyphicon-home"></i> <g:message code="default.home.label"/></a></div>
+				<div class="col-md-2"><g:link action="list"><i class="glyphicon glyphicon-list"></i> <g:message code="default.list.label" args="[entityName]" /></g:link></div>
+				<div class="col-md-2"><g:link action="create"><i class="glyphicon glyphicon-pencil"></i> <g:message code="default.new.label" args="[entityName]" /></g:link></div>
+			</div>
 		</div>
+
+		<div style="margin-top:12px"></div>
+
 		<div id="show-download" class="content scaffold-show" role="main">
 			<h1><g:message code="default.show.label" args="[entityName]" /></h1>
 			<g:if test="${flash.message}">
 			<div class="message" role="status">${flash.message}</div>
 			</g:if>
-			<ol class="property-list download">
-			
+
+			<div class="container-fluid">
```

Comment:
Removed container div.
This sample has both divs: Basic Structure of a Bootstrap Grid (https://www.w3schools.com/bootstrap/bootstrap_grid_system.asp)